packages/next/build/utils.ts

```diff
@@ -84,17 +82,13 @@ export function printTreeView(
             ? 'λ'
             : 'σ'
         } ${item}`,
-        ...(pageInfo
```

Comment:
Should we also simplify the usage of the ⚡character since it seems to mess up alignment? It doesn't look like we could make this character work properly in https://github.com/zeit/next.js/pull/8149

<img width="1025" alt="Screen Shot 2019-11-02 at 12 24 53" src="https://user-images.githubusercontent.com/22380829/68074680-c92cdf80-fd6b-11e9-911f-916b59338beb.png">

Reply:
We'll need to come up with an alternative. Considering there's been numerous releases with these symbols, I don't think it hurts to keep out-of-scope of this PR.

Reply:
Since we're simplifying the output I think it would be good to fix this while we're at it, we could change it to almost anything e.g. `*` or `^` could work since we explain what the symbol correlates to at the bottom of the output